modules/console/pages/config/security/authentication.adoc

```diff
@@ -1,61 +1,357 @@
 = Authentication
-:description: Redpanda Console supports authentication using OAuth 2.0 or OIDC for external identity providers.
-:page-aliases: console:single-sign-on/authentication.adoc, manage:security/console/authentication.adoc
-:page-categories: Management, Security, Redpanda Console
+:description: Authentication in Redpanda Console lets users log in and optionally forward their credentials to the connected Redpanda cluster, ensuring all API requests are executed under the user's identity.
+:page-aliases: console:single-sign-on/authentication.adoc, manage:security/console/authentication.adoc, console:config/security/plain.adoc, console:config/security/okta.adoc, console:config/security/generic-oidc.adoc, console:config/security/keycloak.adoc, console:config/security/github.adoc, console:config/security/google.adoc, console:config/security/azure-ad.adoc
+
+// ========================AUTOMATED TESTS===================================
+// The comments in this file are used to run automated tests of the documented steps. Tests are run using GitHub Actions on each pull request that changes this file in the upstream repository. For more details about the testing tool we use, see https://doc-detective.com/.
+
+// (test start {"id": "console-authentication", "description": "Generate screenshot of Console login screen"})
+// (step {"action":"runShell", "command": "docker compose up -d --wait", "workingDirectory": "../console-login", "timeout": 50000, "exitCodes": [0,1]})
+// (step {"action":"wait", "duration": 20000})
+// (step {"action":"goTo", "url": "http://localhost:8080/login"})
+// (step {"action":"wait", "duration": 10000})
+// (step {"action":"saveScreenshot", "path": "login.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"runShell", "command": "docker compose down -v", "workingDirectory": "../console-login"})
+// (test end)
 
 [NOTE]
 ====
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-Redpanda Console supports authentication using OAuth 2.0 or OIDC for external identity providers, such as:
+This topic describes how to enable authentication in Redpanda Console and explains how it interacts with the Kafka API, Admin API, and Schema Registry API in Redpanda.
+
+Authentication in Redpanda Console lets users log in and optionally forward credentials to the connected cluster, ensuring API requests run under the user's identity. Redpanda Console supports two authentication methods:
+
+* **OIDC (OpenID Connect):**
+  Uses external identity providers, such as Google, to facilitate single sign-on (SSO).
+* **Basic authentication:**
+  Provides a traditional username and password login.
+
+image::console:login.png[]
```

Comment:
Not sure I see the value this image provides--it's the first thing they see, requires nothing complex, and in this context, appears quite large. Thinking we need a set/agreed upon size. Better yet, just show the login fields--the giant Redpanda is superfluous.

Reply:
Cropped the image